src/pending_tasks/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+
+from pending_tasks.api.views import PendingTaskViewSet
+
+router = DefaultRouter()
+router.register(r'api-pending-tasks', PendingTaskViewSet, basename='pending_tasks')
```

Comment:
The url api is wrong. Review underscore and dash symbol and remove 'r' character.